### PR TITLE
Doc: Add structure and content for air-gapped architectures

### PR DESCRIPTION
**Pull request by karenzone**
Fixes: #621 

Air-gapped diagrams were merged with #411.  This work adds structure and content to pull them into the docs.

**PREVIEW:**. https://ingest-docs_bk_654.docs-preview.app.elstc.co/guide/en/ingest/master/use-case-arch.html

### CODE REVIEW

**Comment by karenzone** [comment]
buildkite test this

File: docs/en/ingest-arch/99-airgapped.asciidoc [hunk 1]
```diff
@@ -0,0 +1,15 @@
+[[airgapped-env]]
+== Elastic air-gapped architectures
+
+You can deploy the {stack} with some or all components in a data center or other environment with no access to any outside networks. 
+Some stack components require additional configuration and local dependencies when deployed in air-gapped environments. 
+
+[discrete]
+[[airgapped-archs]]
+=== {agent} to {es} architectures
```
Comment (karenzone) [inline docs/en/ingest-arch/99-airgapped.asciidoc:9]:
```suggestion
=== {agent} to {es} architectures:air-gapped environments
```

**Review by kilfoyle — APPROVED** [review]
LGTM! 🚢

**Comment by karenzone** [comment]
51372

**Pull request merged**